extensions/github-pull-requests/src/PullRequestItem.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+	preferences,
+	List,
+	Icon,
+	ActionPanel,
+	OpenInBrowserAction,
+	CopyToClipboardAction,
+} from "@raycast/api"
+import { PullRequest } from "./types"
+
+export default function PullRequestItem(pullRequest: PullRequest) {
+	const username = preferences.username.value ?? ""
+	const author = pullRequest.author?.login ?? ""
+	const assignees = pullRequest.assignees?.nodes?.map((a) => a.login) ?? []
+	const reviewers =
+		pullRequest.reviewRequests?.nodes?.map((r) => r.requestedReviewer.login) ??
+		[]
+
+	let subtitle = ""
+
+	if (assignees.length > 0) {
+		subtitle += `A: ${assignees.join(", ")}`
+	}
+
+	if (reviewers.length > 0) {
+		if (subtitle.length > 0) {
+			subtitle += " | "
+		}
+
+		subtitle += `R: ${reviewers.join(", ")}`
```

Comment:
I use `  •  ` as a separator in [my GitHub extension](https://github.com/raycast/extensions/blob/81120ca8ddf5d8b44b160232047d5eb9b3f237b5/extensions/github-repository-search/src/utils.ts#L30) which gives a nice visual look.

```suggestion
		subtitle += `R: ${reviewers.join(", ")}`
```

Reply:
Sure, sounds good 👍🏻